site/docs/components/tooltip/index.mdx

```diff
@@ -1,18 +1,18 @@
 ---
 title: Tooltip
 data:
-  description: Tooltip displays a brief message to the user that provides additional information about a UI element. The Tooltip appears after the user’s mouse pointer hovers over the target element for a certain amount of time.  It can communicate new information, errors, warnings, or successful completion of a process or task.
+  description: "`Tooltip` displays a brief message to the user that provides additional information about a user interface (UI) element. The tooltip appears after the user’s mouse pointer hovers over the target element for a certain time.  It can communicate new information, errors, warnings or successful completion of a process or task."
```

Comment:
I believe this is the only time we had (UI) in brackets. Is this a one-off, or should the others be updated?